lib/helper-functions.js

```diff
@@ -1,5 +1,3 @@
-'use strict'
```

Comment:
Intentional change?

Reply:
No, mistake. Good catch!

Reply:
- Re-added the `use strict` (I must have lost it when refactoring helpers 🙄).
- Added doc block